technologies/css.md

```diff
@@ -727,6 +728,7 @@ But in the following example we would not use it:
 [Nature playbook(private)]: https://github.com/nature/playbook
 [bem]: https://github.com/springernature/frontend/practices/bem-css.md
 [a hybrid of OOCSS and Atomic CSS]: https://github.com/springernature/frontend/practices/mosaic-css.md
```

Comment:
Bleurgh. You know what, I'm just gonna delete it entirely. It's unused in the text, so it can go. If somebody needs to refer to it they can add some actual text and then put the link ref back in 😁